src/webgpu/api/validation/shader_module/entry_point.spec.ts

```diff
@@ -15,7 +16,11 @@ TODO:
 `;
```

Comment:
The TODO about unicode normalization is done with the séquençage/se◌́quençage test. It should be removed.

Reply:
Done